int32 object_id
---
srs_msgs/DB_Grasp[] grasp_configuration
